DigitalLearningSolutions.Web/Views/Supervisor/Index.cshtml

```diff
@@ -67,17 +67,28 @@
 <h2>Your to do list</h2>
 @if (Model.SupervisorDashboardToDoItems.Count() > 0)
 {
-  <ul class="nhsuk-contents-list__list">
-    @foreach (var toDoItem in Model.SupervisorDashboardToDoItems)
+<ul class="nhsuk-contents-list__list">
+  @foreach (var toDoItem in Model.SupervisorDashboardToDoItems)
+  {
+
+    if (toDoItem.SignOffRequest)
     {
       <li class="nhsuk-contents-list__item">
         <a class="nhsuk-contents-list__link nhsuk-link--no-visited-state" asp-action="ReviewDelegateSelfAssessment" asp-route-supervisorDelegateId="@toDoItem.SupervisorDelegateId" asp-route-candidateAssessmentId="@toDoItem.ID">
           Review and sign-off @toDoItem.DelegateName's @toDoItem.ProfileName
         </a>. Requested @toDoItem.Requested.ToShortDateString()
-
       </li>
     }
-  </ul>
+    else if(toDoItem.ResultsReviewRequest)
+    {
+      <li class="nhsuk-contents-list__item">
+        <a class="nhsuk-contents-list__link nhsuk-link--no-visited-state" asp-action="VerifyDelegateSelfAssessment" asp-route-supervisorDelegateId="@toDoItem.SupervisorDelegateId" asp-route-candidateAssessmentId="@toDoItem.ID">
+          Verify self-assessment results for @toDoItem.DelegateName's @toDoItem.ProfileName
+        </a>. Requested @toDoItem.Requested.ToShortDateString()
```

Comment:
The ". Requested @toDoItem.Requested..." text falls onto a new blank line if the '<a>' text is long:
![image](https://user-images.githubusercontent.com/67740339/144442405-18f7963e-d91d-456e-b807-600b00fd21c2.png)
Could we fix this to keep it in-line using CSS or move the Requested date inside the link, if not?

Reply:
We should do this for the other type of ToDo item, too.